include/internal/quic_fifd.h

```diff
@@ -46,7 +46,8 @@ struct quic_fifd_st {
     void          (*sstream_updated)(uint64_t stream_id,
                                    void *arg);
     void           *sstream_updated_arg;
-    QLOG           *qlog;
+    QLOG         *(*get_qlog_cb)(void *arg);
```

Comment:
nit: is the indentation off here?

Reply:
Kinda, but it's more consistent with what's already being done above. (I wrote this before it came to my attention we don't dangle pointer sigils like this. But seems better than just bloating the diff.)